website/src/examples/point-cloud-layer.js

```diff
@@ -35,7 +37,7 @@ class PointCloudDemo extends Component {
 
   render() {
     return <div style={{width: '100%', height: '100%', background: '#ecdbce'}}>
-      <App onLoad={this._onLoad} />
+      <App onLoad={this._onLoad} device={this.props.device} />
```

Comment:
We should trigger React to create a new App when device changes. Maybe it can be done by setting a key?
```suggestion
      <App key={device.type} onLoad={this._onLoad} device={this.props.device} />
```

Reply:
i suppose this depends on the above comment around whether we should handle re-initialising on a device prop change, happy to implement this if the consensus is to not reinitialise, but it does feel odd to me to not handle a prop changing.